src/thok.rs

```diff
@@ -68,6 +71,33 @@ impl Thok {
         }
     }
 
+    pub fn save_results(&self) -> io::Result<()> {
+        let log_path = dirs::data_dir().unwrap().join("thokr.log");
+        // If the config file doesn't exist, we need to emit a header
+        let needs_header = !log_path.exists();
+
+        let mut log_file = OpenOptions::new()
+            .write(true)
+            .append(true)
+            .create(true)
+            .open(log_path)?;
+
+        if needs_header {
+            writeln!(log_file, "# Date, WPM, Accuracy, Standard Deviation")?;
```

Comment:
Not a big fan of `# Date` -- mostly just the `#` feels confusing. I'd want to add a few more headers here too, namely to track what parameters the test was run with. Was it 15 words? 100? Was it a 30 second test? 60? Stuff like that!

Reply:
I'm happy to add that for sure.  I'm not the most familiar with the project, so what might be the best way to get the word count other than counting the `' '` chars in `self.input`?